<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assignment3QtClass</class>
 <widget class="QMainWindow" name="Assignment3QtClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1389</width>
    <height>922</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assignment3Qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>10</y>
      <width>161</width>
      <height>901</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_RW">
       <property name="text">
        <string>ResolutionW</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="resolutionW">
       <property name="text">
        <string>400</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_RH">
       <property name="text">
        <string>ResolutionH</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="resolutionH">
       <property name="text">
        <string>300</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_AA">
       <property name="text">
        <string>AntiAliasing</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="antiAliasing">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_ICR">
       <property name="text">
        <string>ImageScale</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="imageScaleRatio">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_CP">
       <property name="text">
        <string>CameraPos</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="CameraPos">
       <property name="text">
        <string>0.0, 1.0, 10.0</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_LA">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>LookAt</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="CameraLookAt">
       <property name="text">
        <string>0.0, 0.0, 0.0</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_SD">
       <property name="text">
        <string>sceneData</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QPushButton" name="pushButton_Browse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="2">
      <widget class="QLineEdit" name="sceneDataPath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>../sceneData.txt</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="2">
      <widget class="QLabel" name="label_TValue">
       <property name="text">
        <string>Time: 0s</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="14" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_Render">
       <property name="text">
        <string>Render</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>540</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_Image">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1200</width>
      <height>900</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Result Image</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>resolutionW</tabstop>
  <tabstop>resolutionH</tabstop>
  <tabstop>antiAliasing</tabstop>
  <tabstop>imageScaleRatio</tabstop>
  <tabstop>CameraPos</tabstop>
  <tabstop>CameraLookAt</tabstop>
  <tabstop>pushButton_Browse</tabstop>
  <tabstop>sceneDataPath</tabstop>
  <tabstop>pushButton_Render</tabstop>
 </tabstops>
 <resources>
  <include location="assignment3qt.qrc"/>
 </resources>
 <connections/>
</ui>
